### DISCUSSION TITLE
ControlCatalog.Web

### DISCUSSION THREAD

**Discussion by TSerious** [AvaloniaUI/Avalonia#7949] [Q&A] [answered]
Hello,
I am trying to build the sample ControlCatalog.Web on Windows 10 with VS2022 but I get the error that libSkiaSharp.dll couldn't be found. This goes for the latest commit in the master branch (commit 6e359707). The commit acf176a8 in the 0.10.13-fork-fix works well.
Thanks for the help. The ControlCatalog.Desktop application works well.

**Comment by maxkatz6** [comment]
Install warm workload
Something like this
`dotnet workload install wasm-tools`

**Reply by TSerious** [reply]
Thanks for the help, but this didn't solve it.
Still: System.DllNotFoundException: libSkiaSharp (in blazor.webassembly.js:1)
when running the ControlCatalog.Web project

**Reply by maxkatz6** [reply]
Double check if you have latest .NET 6 SDK. And try to delete workloads and install again.

Also see https://docs.avaloniaui.net/tutorials/running-in-the-browser

**Reply by TSerious** [reply]
I' still getting System.DllNotFoundException: libSkiaSharp.
I reinstalled the latest .Net6 (6.0.3), deleted workloads and installed them again.
Here is my dotnet --info result:
`.NET SDK (reflecting any global.json):
 Version:   6.0.201
 Commit:    ef40e6aa06

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.201\

Host (useful for support):
  Version: 6.0.3
  Commit:  c24d9a9c91

.NET SDKs installed:
  5.0.301 [C:\Program Files\dotnet\sdk]
  6.0.201 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.28 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.7 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]`

Workload Microsoft.WebAssembly.Sdk.6.0.3-x64 is installed. I also deleted and repaired it:
`C:\Users\tober>dotnet workload repair

Repairing workload installation for workloads: wasm-tools
Repairing Microsoft.WebAssembly.Sdk.6.0.3-x64.msi ....... Done
Repairing Microsoft.Mono.browser-wasm.6.0.3-x64.msi ........ Done
Repairing Microsoft.AOT.win-x64.Cross.browser-wasm.6.0.3.msi .... Done
Repairing Microsoft.MonoAOTCompiler.Task.6.0.3-x64.msi .... Done
Repairing Microsoft.MonoTargets.Sdk.6.0.3-x64.msi ..... Done
Repairing Microsoft.Emscripten.2.0.23.Node.win-x64.6.0.2.msi ..... Done
Repairing Microsoft.Emscripten.2.0.23.Python.win-x64.6.0.2.msi ............ Done
Repairing Microsoft.Emscripten.2.0.23.Sdk.win-x64.6.0.2.msi ................................................................ Done

Successfully repaired workloads: wasm-tools`

**Comment by TSerious** [comment, marked as answer]
This was a bug and is fixed with: [https://github.com/AvaloniaUI/Avalonia/pull/7962](https://github.com/AvaloniaUI/Avalonia/pull/7962)
